www/source/blog/2019-03-04-introducing-the-install-hook.html.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Introducing the Install Hook!
```

Comment:
I love that there's a blog post. However, if it's part of this PR, it will be published when this merges, which could be confusing unless we word things properly (or do we have some awareness of dates that embargoes things until the right time?)

Reply:
Yeah I was thinking we keep the ridiculous future date and then update the date on release.